operators/src/opencl/cl_program.rs

```diff
@@ -404,31 +429,61 @@ impl<'a> CLProgramRunnable<'a> {
                         // no geo
                     }
                     TypedFeatureCollection::MultiPoint(points) => {
-                        let coordinates = points.coordinates();
-                        let buffer = Buffer::<Double2>::builder()
-                            .queue(queue.clone())
-                            .len(coordinates.len())
-                            .copy_host_slice(unsafe {
-                                std::slice::from_raw_parts(
-                                    coordinates.as_ptr() as *const Double2,
-                                    coordinates.len(),
-                                )
-                            })
-                            .build()?;
-
-                        kernel.set_arg(format!("IN_POINT_COORDS{}", idx), &buffer)?;
-
-                        let coordinates_offsets = points.multipoint_offsets();
-                        let buffer = Buffer::builder()
-                            .queue(queue.clone())
-                            .len(coordinates_offsets.len())
-                            .copy_host_slice(coordinates_offsets)
-                            .build()?;
-
-                        kernel.set_arg(format!("IN_POINT_OFFSETS{}", idx), &buffer)?;
+                        kernel.set_arg(
+                            format!("IN_FEATURE{}_COORDS", idx),
```

Comment:
maybe call it `IN_COLLECTION{}` because its not one feature but a collection?

Reply:
or plural: features

Reply:
b9b625f